recipes/python/pyyaml.yaml

```diff
@@ -0,0 +1,9 @@
+inherit: [ python3-pkg ]
```

Comment:
I think this should be `python3-cext-pkg`. PyYAML provides some cpython extension to improve parsing speed. Hopefully this is working.

Reply:
done